src/content/docs/zh-cn/guides/view-transitions.mdx

```diff
@@ -233,18 +233,82 @@ const myFade = {
 
 <header transition:animate={myFade}> ... </header>
 ```
+## 路由控制
 
-## 阻止客户端导航
+`<ViewTransitions />` 路由通过监听以下事件来处理导航：
 
-一旦你在页面上添加了 `<ViewTransitions />` 组件，就可以通过客户端路由导航到页面上链接到站点上的另一个页面的每个锚点。有一些情况下，你可能不希望通过 CSR 导航。一个例子是，如果链接指向任何非页面，例如 `public/` 文件夹中的 PDF。或者是一个生成图像的 API 路由。
+- 点击 `<a>` 元素。
+- 后退和前进导航事件。
 
-在这些情况下，你可以使用 `data-astro-reload` 属性来选择性地阻止每个链接使用客户端路由，如下所示：
+如下选项允许你进一步控制在路由中发生的导航事件：
 
-```html
-<a href="/quarterly-earnings.pdf" data-astro-reload>
+-  `astro-data-reload`：一个 `<a>` 标签属性，用于 [进行强制全页面导航](#阻止客户端导航)。
+-  `astro-data-history="auto | push | replace"`：一个 `<a>` 标签属性，用于 [控制浏览器的历史记录](#替换浏览器历史条目)。
+-  `navigate(href, options)`：任何客户端脚本或客户端组件可用的方法，用于[触发导航](#阻止客户端导航)。
+
+### 阻止客户端导航
+
+由于关联的两个页面须使用 `<ViewTransitions />` 路由以防止重新加载全页面，因此在某些情况下你无法通过 CSR 进行导航。你可能也不希望在每次导航更改时都使用客户端路由，而是更希望在特定路由上进行传统的页面导航。
```

Comment:
> There are some cases where you cannot navigate via CSR since both pages involved must use the `<ViewTransitions />` router to prevent a full-page reload.

```suggestion
只有关联的两个页面都使用了 `<ViewTransitions />` 路由才能通过 CSR (client-side routing，客户端路由) 进行导航，防止重新加载全页面。你可能也不希望在每次导航更改时都使用客户端路由，而是更希望在特定路由上进行传统的页面导航。
```